package.json

```diff
@@ -277,6 +277,11 @@
           "default": true,
           "markdownDescription": "Show section parameter type. Default is `true`."
         },
+        "qute.inlayHint.showSectionParameterType": {
```

Comment:
This key should be `qute.inlayHints.showSectionParameterDefaultValue` (currently there are two entries for `qute.inlayHint.showSectionParameterType`)

Reply:
Ooops sory, I forget to push my changed, it should be better